src/dmd/backend/dwarfdbginf.d

```diff
@@ -87,13 +84,11 @@ static if (ELFOBJ || MACHOBJ)
 
     __gshared
     {
-        static if (MACHOBJ)
-        {
+        //static if (MACHOBJ)
             int except_table_seg = UNKNOWN; // __gcc_except_tab segment
             int except_table_num = 0;       // sequence number for GCC_except_table%d symbols
             int eh_frame_seg = UNKNOWN;     // __eh_frame segment
             Symbol *eh_frame_sym = null;    // past end of __eh_frame
-        }
 
```

Comment:
Just remove the static if ?

Reply:
It's a reasonable way to say that those declarations are only for Mach.